docs/content/en/docs/Reference/image_matrix.md

```diff
@@ -37,7 +37,7 @@ Base images are tagged with specific upstream versions (e.g. Ubuntu 20 LTS is pi
 | **Ubuntu based (rolling)** **                                   	       | https://quay.io/repository/kairos/core-ubuntu           	               | https://quay.io/repository/kairos/kairos-ubuntu           	               |
 | **Ubuntu based (22 LTS)** **                                    	       | https://quay.io/repository/kairos/core-ubuntu-22-lts           	       | https://quay.io/repository/kairos/kairos-ubuntu-22-lts           	       |
 | **Ubuntu based (20 LTS)** **                                   	         | https://quay.io/repository/kairos/core-ubuntu-20-lts           	       | https://quay.io/repository/kairos/kairos-ubuntu-20-lts           	       |
-| **Rocky Linux based**                                                    | https://quay.io/repository/kairos/core-rockylinux                       | N/A                                                                       |
+| **Rocky Linux based**                                                    | https://quay.io/repository/kairos/core-rockylinux                       | https://quay.io/repository/kairos/kairos-rockylinux                       |
```

Comment:
![image](https://user-images.githubusercontent.com/2794419/220304834-d4c771e2-12c0-4986-b8e5-fad07a4b9258.png)

Reply:
Yeah, it's pending the master build from kairos-io/provider-kairos#241 which was just merged so we should have it some time today :crossed_fingers:

Reply:
I'll wait a few hours before merging